migrations/versions/0411_broadcast_message_history.py

```diff
@@ -0,0 +1,74 @@
+"""
+
+Revision ID: 0411_broadcast_message_history
+Revises: 5206923870d3
+Create Date: 2025-01-30 10:00:00
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = "0411_broadcast_message_history"
+down_revision = "5206923870d3"
+
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy.dialects import postgresql
+
+
+def upgrade():
+    op.execute("delete from broadcast_message where status = 'draft'")
+    op.create_table(
+        "broadcast_message_history",
+        sa.Column("id", postgresql.UUID(as_uuid=True), nullable=False),
+        sa.Column("reference", sa.String(length=255), nullable=False),
+        sa.Column("created_at", sa.DateTime(), nullable=False),
+        sa.Column("content", sa.Text(), nullable=False),
+        sa.Column("service_id", postgresql.UUID(as_uuid=True), nullable=False),
+        sa.Column("created_by_id", postgresql.UUID(as_uuid=True), nullable=True),
+        sa.Column("version", sa.Integer(), autoincrement=False, nullable=False),
+        sa.Column("areas", postgresql.JSONB(none_as_null=True, astext_type=sa.Text())),
+        sa.Column("duration", sa.Interval(), nullable=True),
+        sa.PrimaryKeyConstraint("id", "version"),
```

Comment:
You could either create a specific id column with an autogenerated uuid (after renaming id to broadcast_message_id) or have a compound key on broadcast_message_id and created_at (though that's more fragile in case 2 people save in the same second).